layers/meta-opentrons/recipes-images/images/opentrons-ot3-image.bb

```diff
@@ -35,9 +35,15 @@ IMAGE_INSTALL += " \
     robot-app-wayland-launch robot-app \
     opentrons-robot-server opentrons-update-server \
     python3 python3-misc python3-modules \
+    opentrons-usb-bridge \
```

Comment:
we also have to add this to the DEPENDS variable above
`DEPENDS += "rsync-native zip-native opentrons-robot-server opentrons-update-server"`